Readme.md

```diff
@@ -5,7 +5,18 @@
 node-prune is a small tool to prune unnecessary files from ./node_modules, such as markdown, typescript source files, and so on. Primarily built for [Up](https://github.com/apex/up) which lets you deploy serverless web applications in seconds.
 
 ## Installation
+If Go is already installed check check if your are using latest version, comparing to https://github.com/golang/go/releases, type this command to check your local Go version 
 
+```
+$ go version
+```
+Before install package make sure your GOPATH in` ~/.bash_profile` is set. It must be path to Go binaries. 
```

Comment:
Make it shell-agnostic. Not everyone uses `bash`. Also don't forget Windows users, albeit the installer for Go on Windows __should__ add Go to the `PATH`.

Reply:
Before _installing_ [the] package.